modules/timeDate.js

```diff
@@ -0,0 +1,11 @@
+import { DateTime } from '../node_modules/luxon/src/luxon.js';
+
+const time = document.querySelector('.time-date');
+
+const setTimeDate = () => {
+  const dt = DateTime.now();
+  const localTime = dt.toLocaleString(DateTime.DATETIME_MED);
+  time.textContent = localTime;
+};
+
+export default setTimeDate;
```

Comment:
- [ ] It is required the current date is displayed below the navigation bar. Kindly fix that.

### Your Website
![date](https://github.com/Yonashenok/Awesome-Books-ES6/assets/26597432/37b247bc-e836-44c8-a724-25eefa6176ad)

### Project Requirement
![project-rqt](https://github.com/Yonashenok/Awesome-Books-ES6/assets/26597432/d8cf8492-7681-4e47-ac3c-345669e6e897)